openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/common/azureConfig.json

```diff
@@ -1,15 +0,0 @@
-{
-    "$id": "https://open-metadata.org/schema/entity/services/connections/database/common/azureConfig.json",
-    "$schema": "http://json-schema.org/draft-07/schema#",
-    "title": "Azure Configuration Source",
-    "description": "Azure Database Connection Config",
-    "type": "object",
-    "javaType": "org.openmetadata.schema.services.connections.database.common.AzureConfig",
-    "properties": {
-        "azureConfig": {
-            "title": "Azure Credentials Configuration",
-            "$ref": "../../../../../security/credentials/azureCredentials.json"
-        }
-    },
-    "additionalProperties": false
-}
```

Comment:
@mgorsk1 we can not remove this file as this is being referred by other services as well for example mysql: https://github.com/open-metadata/OpenMetadata/blob/fa66b5580654e2187aee31bc5611574ed2949b4c/openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/mysqlConnection.json#L55

@ayush-shah maybe can you explain the purpose of creating this new file instead of referring the `security/credentials/azureCredentials.json` directly?

Reply:
I can revert, the reason i refactored was AzureCredentials did not show as an option in the ui when selecting authentication mechanism.